example/fastlane/Fastfile

```
@@ -34,3 +34,44 @@ platform :ios do
+    )
+
+    upload_to_play_store(
+      track: "internal",
+      release_status: "completed",
+      json_key_data: ENV["PLAY_STORE_JSON_KEY"],
+      aab: "android/app/build/outputs/bundle/release/app-release.aab",
+      skip_upload_apk: true,
+      skip_upload_metadata: true,
+      skip_upload_changelogs: true,
+      skip_upload_images: true,
+      skip_upload_screenshots: true,
```

RuboCop (1.84.2)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)